bevy_srv/salt/developer/golang.sls

```diff
@@ -0,0 +1,50 @@
+---
+# Salt state for installing go https://golang.org/doc/install
+#
+{% set go_vers = salt['pillar.get']('golang_version', '1.13.7') %} {# TODO: update this default regularly #}
+
+{% if grains['os'] == 'Windows' %}
+include:
+  - windows.install_chocolatey
+
+golang:
+  chocolatey.upgraded:
```

Comment:
Should we include a version number in the Mac/Windows installations or just Linux?

Reply:
I only included in for Linux, because, it appears, that you MUST supply it for Linux. I will supply an update documenting the inconsistency, and add a TODO to support compiler version control.